.github/workflows/installer.yml

```diff
@@ -23,8 +23,7 @@ jobs:
       fail-fast: false
       matrix:  # just a vector in this case
         make_env: [
-          IPC_VERSION=,  # default version
-          IPC_VERSION=IPC4 UNSIGNED_list= SIGNED_list='tgl tgl-h',
+          IPC_VERSION= UNSIGNED_list='imx8' SIGNED_list=,  # default version
```

Comment:
@dbaluta, @andyross , is anyone outside Intel looking at IPC4? With or without Zephyr?

Reply:
@bhiregou @bhiregoudar @yaochunhung

Reply:
Right now i.MX8 IPC4 support is a discussion topic. I am not committing to doing it soon, BUT it is in my backlog, I am just lacking some information related to the firmware side of this conversion (for the kernel I have some patches ready, minus exact filenames)

I do expect Zephyr support to be done before the IPC4 migration.

Reply:
> I do expect Zephyr support to be done before the IPC4 migration.

Thanks, this is the information relevant to this line. It means this Github Action will never need to build IPC4+XTOS ever again so this line can be removed as @lyakh currently does.